taca/utils/bioinfo_tab.py

```diff
@@ -173,22 +173,17 @@ def get_ss_projects(run_dir):
         lanes = str(1)
         #Pattern is a bit more rigid since we're no longer also checking for lanes
         sample_proj_pattern=re.compile("^((P[0-9]{3,5})_[0-9]{3,5})$")
-        data = parse_samplesheet(FCID_samplesheet_origin, run_dir)
-    #Hiseq X case
-    elif "Hiseq X" in runtype: 
+        data = parse_samplesheet(FCID_samplesheet_origin, run_dir, is_miseq=True)
+    #HiSeq X case
+    elif "HiSeq X" in runtype:
         FCID_samplesheet_origin = os.path.join(CONFIG['bioinfo_tab']['xten_samplesheets'],
                                     current_year, '{}.csv'.format(FCID))   
         data = parse_samplesheet(FCID_samplesheet_origin, run_dir)
-    #Hiseq 2.5k case
-    elif "Hiseq" or "TruSeq" in runtype:
+    #HiSeq 2500 case
+    elif "HiSeq" in runtype or "TruSeq" in runtype:
         FCID_samplesheet_origin = os.path.join(CONFIG['bioinfo_tab']['hiseq_samplesheets'],
                                     current_year, '{}.csv'.format(FCID)) 
-        try:
-            csvf=open(FCID_samplesheet_origin, 'rU')
-            data=DictReader(csvf)
-        except:
-            logger.warn("Cannot initialize DictReader for {}. Most likely due to poor comma separation".format(run_dir))
-            return []
+        data = parse_samplesheet(FCID_samplesheet_origin, run_dir)
```

Comment:
Always assumed the parse_samplesheet class couldn't handle HiSeq 2.5k since DictReader has been used since the creation of the script. But if it works it works.